java/code/src/com/redhat/rhn/domain/image/ImageFile.java

```diff
@@ -19,43 +19,39 @@
 
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.hibernate.annotations.Type;
 
-import java.util.HashSet;
-import java.util.Set;
-
-import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
-import javax.persistence.OneToMany;
 import javax.persistence.SequenceGenerator;
 import javax.persistence.Table;
 
 /**
- * ImageBuildHistory
+ * ImageFile
  */
 @Entity
-@Table(name = "suseImageBuildHistory")
-public class ImageBuildHistory extends BaseDomainHelper {
+@Table(name = "suseImageFile")
+public class ImageFile extends BaseDomainHelper {
 
     private Long id;
-    private int revisionNumber;
     private ImageInfo imageInfo;
-    private Set<ImageRepoDigest> repoDigests = new HashSet<>();
+    private String file;
+    private String type;
+    private boolean external;
 
     /**
      * @return the id
      */
     @Id
     @Column(name = "id")
-    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "imgbuildhistory_seq")
-    @SequenceGenerator(name = "imgbuildhistory_seq",
-            sequenceName = "suse_img_buildhistory_id_seq")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "imgfile_seq")
+    @SequenceGenerator(name = "imgfile_seq",
+            sequenceName = "suse_img_file_id_seq")
```

Comment:
sequence name should follow the existing naming convention: `suse_image_file_id_seq`

Reply:
fixed